demos/calibration/calibration-pubmed-node-classification.ipynb

```diff
@@ -24,11 +24,11 @@
    "source": [
     "This notebook demonstrates probability calibration for multi-class node attribute inference. The classifier used is GraphSAGE and the dataset is the citation network Pubmed-Diabetes. Our task is to predict the subject of a paper (the nodes in the graph) that is one of 3 classes. The data are the network structure and for each paper a 500-dimensional TF/IDF word vector.\n",
     "\n",
-    "The notebook demonstrates the use of `stellargraph`'s `TemperatureCalibration` and `IsotonicCalibration` classes as well as supporting methods for calculating the Expected Calibration Error (ECE) and plotting reliability diagrams.\n",
+    "The notebook demonstrates the use of `StellarGraph`'s `TemperatureCalibration` and `IsotonicCalibration` classes as well as supporting methods for calculating the Expected Calibration Error (ECE) and plotting reliability diagrams [2].\n",
```

Comment:
Only minor thing; if we're referring to the StellarGraph python module as it's imported, then `` `stellargraph` `` as it was written is correct, but if we're referring to it as the whole library/product, then it probably shouldn't be code-quoted:

```suggestion
    "The notebook demonstrates the use of StellarGraph's `TemperatureCalibration` and `IsotonicCalibration` classes as well as supporting methods for calculating the Expected Calibration Error (ECE) and plotting reliability diagrams [2].\n",
```